apps/web/composables/useProducts/useProducts.ts

```diff
@@ -37,7 +37,8 @@ export const useProducts: UseProductsReturn = (category = '') => {
     state.value.loading = true;
 
     if (params.categoryUrlPath?.endsWith('.js')) return state.value.data;
-    const identifier = category || params.categoryUrlPath || params.categoryId;
```

Comment:
Andrei is working on a fix here. We need all params, it also needs to respect the sorting and so on.